src/sidebar/store/modules/defaults.js

```diff
@@ -55,6 +55,17 @@ function getDefaults(state) {
   return state;
 }
 
+/**
+ * @typedef DefaultsStore
+ *
+ * // Actions
+ * @prop {typeof setDefault} setDefault
+ *
+ * // Selectors
+ * @prop {(key: string) => string|null} getDefault
+ * @prop {() => Object.<string,null>} getDefaults
```

Comment:
```suggestion
 * @prop {() => Object.<string,string|null>} getDefaults
```

The current type means that `getDefaults` values are always `null`, whereas they should be `string|null`.

Reply:
yep, that was my question above https://github.com/hypothesis/client/pull/2370#discussion_r459461194

this makes more sense :)